utils/lru.c

```diff
@@ -22,6 +24,31 @@ typedef struct Entry {
   struct list_head list;
 } Entry;
 
+
+void
+glusterBlockUpdateLruLogdir(const char *logPath)
+{
+  struct list_head *pos, *q;
+  Entry *tmp;
+
+
+  LOCK(lru_lock);
+  if (!logPath || list_empty(&Cache)) {
+      UNLOCK(lru_lock);
+      return;
+  }
+
+  list_for_each_safe(pos, q, &Cache){
+    tmp = list_entry(pos, Entry, list);
+    if (glfs_set_logging(tmp->glfs, logPath, GFAPI_LOG_LEVEL)) {
+      LOG("mgmt", GB_LOG_WARNING, "glfs_set_logging(%s, %d) on %s failed[%s]",
+          logPath, GFAPI_LOG_LEVEL, tmp->volume, strerror(errno));
```

Comment:
May be later, it make more sense to covert GFAPI_LOG_LEVEL as configurable.

Reply:
Yeah, If you agree, let's do it in late future.
Thanks.